jac/jaclang/tests/test_cli.py

```diff
@@ -548,3 +548,14 @@ def test_caching_issue(self) -> None:
                 self.assertNotIn("Passed successfully.", stdout)
                 self.assertIn("F", stderr)
         os.remove(test_file)
+
+    def test_node_del(self) -> None:
+        """Test for graph CLI cmd."""
+        captured_output = io.StringIO()
+        sys.stdout = captured_output
+        cli.dot(f"{self.fixture_abs_path('node_del.jac')}")
+        sys.stdout = sys.__stdout__
+        with open("node_del.dot") as f:
```

Comment:
Need to clean up this file (or add a to_screen option in dot command and capture stdio).